packages/abi-gen/test-cli/output/typescript/abi_gen_dummy.ts

```diff
@@ -2124,46 +2214,46 @@ export class AbiGenDummyContract extends BaseContract {
          * to create a 0x transaction (see protocol spec for more details).
          * @returns The ABI encoded transaction data as a string
          */
-        getABIEncodedTransactionData(): string {
+        getABIEncodedTransactionData(index_0: BigNumber): string {
+            assert.isBigNumber('index_0', index_0);
             const self = (this as any) as AbiGenDummyContract;
-            const abiEncodedTransactionData = self._strictEncodeArguments('methodReturningMultipleValues()', []);
+            const abiEncodedTransactionData = self._strictEncodeArguments('simpleInputSimpleOutput(uint256)', [
+                index_0,
+            ]);
             return abiEncodedTransactionData;
         },
         /**
          * Decode the ABI-encoded transaction data into its input arguments
          * @param callData The ABI-encoded transaction data
          * @returns An array representing the input arguments in order. Keynames of nested structs are preserved.
          */
-        getABIDecodedTransactionData(callData: string): void {
+        getABIDecodedTransactionData(callData: string): BigNumber {
             const self = (this as any) as AbiGenDummyContract;
-            const abiEncoder = self._lookupAbiEncoder('methodReturningMultipleValues()');
+            const abiEncoder = self._lookupAbiEncoder('simpleInputSimpleOutput(uint256)');
             // tslint:disable boolean-naming
-            const abiDecodedCallData = abiEncoder.strictDecode<void>(callData);
+            const abiDecodedCallData = abiEncoder.strictDecode<BigNumber>(callData);
             return abiDecodedCallData;
         },
         /**
          * Decode the ABI-encoded return data from a transaction
          * @param returnData the data returned after transaction execution
          * @returns An array representing the output results in order.  Keynames of nested structs are preserved.
          */
-        getABIDecodedReturnData(returnData: string): [BigNumber, string] {
+        getABIDecodedReturnData(returnData: string): BigNumber {
             const self = (this as any) as AbiGenDummyContract;
-            const abiEncoder = self._lookupAbiEncoder('methodReturningMultipleValues()');
+            const abiEncoder = self._lookupAbiEncoder('simpleInputSimpleOutput(uint256)');
             // tslint:disable boolean-naming
-            const abiDecodedReturnData = abiEncoder.strictDecodeReturnValue<[BigNumber, string]>(returnData);
+            const abiDecodedReturnData = abiEncoder.strictDecodeReturnValue<BigNumber>(returnData);
             return abiDecodedReturnData;
         },
     };
-    public methodReturningArrayOfStructs = {
+    public simplePureFunction = {
```

Comment:
Is the order of methods in the generated wrapper non-deterministic?  That would be really unfortunate... It seems like there's a ton of "noise" in this diff due simply to a re-ordering of method declarations.  If that's the case, we should consider sorting them before generating them.  Out of scope for this PR of course, but maybe we should make an Issue about it.

Reply:
see https://github.com/0xProject/0x-monorepo/pull/2243#discussion_r332724650